Samples/Controllers/NorthwindController.cs

```diff
@@ -137,10 +126,8 @@ orderby o.OrderDate
 
         [HttpGet]
         public object SalesByCategory() {
-            // NOTE see the #warning at the top of the file
-            var temp = _nwind.Order_Details.Include(d => d.Product.Category).ToArray();
-
-            return from d in temp
+#warning TODO remove ToArray() when https://github.com/aspnet/EntityFramework/issues/3676 fix is released
+            return from d in _nwind.Order_Details.Include(d => d.Product.Category).ToArray()
```

Comment:
@AlekseyMartynov ,

Why do we need call the "ToArray()" method here? What difference comparing with "SalesByCategoryYear" method implementation?